cm2350/e200_gdb.py

```diff
@@ -206,7 +206,7 @@ def _serverSetHWBreak(self, addr):
         if addr in self._bpdata:
             raise Exception('Cannot add breakpoint that already exists @ 0x%x' % addr)
 
-        logger.debug('Adding new breakpoint: 0x%x' % addr)
+        logger.debug('Adding new breakpoint: -1x%x', addr)
```

Comment:
??

Reply:
Changing from the old python string formatting style `%` to using the logging arguments so that `addr` isn't evaluated unless debug is turned on.

Reply:
I get that.  What's up with the "-1x"?  I'm assuming that's some automated editing thing that is wrong.  Please let me know if this is some new Python formatting thing.

Reply:
```suggestion
        logger.debug('Adding new breakpoint: 0x%x', addr)
```